en/theme/material/main.html

```diff
@@ -1,71 +1,24 @@
-<!--
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- * WSO2 Inc. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
--->
+<!-- #Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
LLC ?

Reply:
Fixed in https://github.com/wso2/docs-ballerina-vscode-extension/pull/183#discussion_r1203382727.